src/pages/home/report/ReportActionsView.js

```diff
@@ -66,9 +66,11 @@ function ReportActionsView(props) {
     const reactionListRef = useContext(ReactionListContext);
     const didLayout = useRef(false);
     const didSubscribeToReportTypingEvents = useRef(false);
-    const hasCachedActions = useRef(_.size(props.reportActions) > 0);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    const hasCachedActions = useMemo(() => _.size(props.reportActions) > 0, []);
 
-    const mostRecentIOUReportActionID = useRef(ReportActionsUtils.getMostRecentIOURequestActionID(props.reportActions));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    const mostRecentIOUReportActionID = useMemo(() => ReportActionsUtils.getMostRecentIOURequestActionID(props.reportActions), []);
```

Comment:
:warning: Now that you've converted these to `useMemo`, we'll also need to update their usages since they still reference `hasCachedActions.current` and `mostRecentIOUReportActionID.current`.